db/bigtable.go

```diff
@@ -856,7 +862,12 @@ func (bigtable *Bigtable) GetValidatorSyncDutiesHistory(validators []uint64, sta
 	defer cancel()
 
 	rangeStart := fmt.Sprintf("%s:e:%s:s:", bigtable.chainId, reversedPaddedEpoch(startEpoch))
-	rangeEnd := fmt.Sprintf("%s:e:%s:s:", bigtable.chainId, reversedPaddedEpoch(startEpoch-uint64(limit)))
+	var rangeEnd string
+	if startEpoch >= uint64(limit) {
+		rangeEnd = fmt.Sprintf("%s:e:%s:s:", bigtable.chainId, reversedPaddedEpoch(startEpoch-uint64(limit)))
+	} else {
+		rangeEnd = fmt.Sprintf("%s:e:%s:s:%s", bigtable.chainId, reversedPaddedEpoch(0), "\x00") // add \x00 to rangeEnd so epoch 0 is included
```

Comment:
From my understanding this will always return an empty array as the rangeEnd will be smaller than the rangeStart. I don't think a query where limit > startEpoch makes sense and I would suggest to return an error instead. Especially, considering that we currently cannot query for epoch 0.

Reply:
It might be a good Idea to return an error here instead of running a query that will always return a zero length array.